examples/computer_vision/_mnist_pl/mnist.py

```diff
@@ -0,0 +1,94 @@
+import torch
+import pytorch_lightning as pl
+from torch.utils.data import DataLoader
+from torch.nn import functional as F
+from torchvision import datasets, transforms
+from torchvision.datasets import MNIST
+from pathlib import Path
+from typing import Optional
+import os
+
+
+class LightningMNISTClassifier(pl.LightningModule):
+
+    def __init__(self, *args, lr: float, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        self.save_hyperparameters()
```

Comment:
So that the inputs get saved under `self.hparams` if I'm not reading this wrong https://pytorch-lightning.readthedocs.io/en/stable/common/lightning_module.html#save-hyperparameters